packages/promotion/src/models/application-method.ts

```diff
@@ -51,6 +54,13 @@ export default class ApplicationMethod {
   })
   promotion: Promotion
 
+  @ManyToMany(() => PromotionRule, "application_methods", {
+    owner: true,
+    pivotTable: "application_method_promotion_rule",
+    cascade: ["soft-remove"] as any,
+  })
+  target_rules = new Collection<PromotionRule>(this)
```

Comment:
promotion_rules might make it more vague. If you feel strongly about this, feel free to open a discussion [here](https://www.notion.so/medusajs/Promotions-d865e4430d544b89abffba2e3e9a721f) so that everyone is aware of it.

Reply:
I ve put a comment, it is non blocking in any case :) let's move like this and see later

Reply:
There is a subtle difference between the rules and target rules. Rules specify the eligibility of the promotion, and target rules specify, if eligible, what the promotion applies to. 

E.g. "The promotion giving you 50% off is valid if the customer is VIP, but only on products Y and Z". 

"customer is VIP" -> `rules`
"only on products Y and Z" -> `target_rules`

Imo. using rules for both would leave us with more ambiguity around the concept. Although the meaning of `target_rules` might not be immediately obvious, it at least indicates that the rules are different from the ones at the promotion level.

Reply:
Thanks for the explanation guys, i ve missed this subtlety 😅 so there is the rule of the promotion and to what they can be applicable 👌